packages/daimo-api/src/server/router.ts

```diff
@@ -576,6 +578,21 @@ export function createRouter(
         return status.isValid;
       }),
 
+    getBinanceWithdrawalURL: publicProcedure
```

Comment:
idea:

```ts
getExchangeURL: publicProcedure
      .input(
        z.object({
          addr: zAddress,
          platform: z.enum(["ios", "android", "other"]),
          exchange: z.enum(["binance"]),
          direction: z.enum(["depositFromExchange", "withdrawToExchange"]),
        })
      )
      .query(async (opts) => {
        const { addr, platform, exchange, direction } = opts.input;
        const acc = nameReg.getDaimoAccount(addr);
        if (!acc) throw new TRPCError({ code: "NOT_FOUND" });

        switch(`${exchange}-${direction}`) {
          case "binance-depositFromExchange":
            return await binanceClient.createWithdrawalURL(addr, platform);
          default:
            throw new TRPCError({ code: "NOT_FOUND" });
        }
      }),
```

...easier to add Lemon etc in future without lots of API methods

because of old app versions, once we add an API method, we can't remove it for a long time

Reply:
done, refactored to more generality, added sort order also

<img width="522" alt="image" src="https://github.com/daimo-eth/daimo/assets/6984346/ef1bcf1a-c8bc-43aa-b310-c50934fd9b9e">